### PR TITLE
graph-tool 2.41

### PR DESCRIPTION
**Pull request by count0**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/graph-tool.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@ class GraphTool < Formula
 
   desc "Efficient network analysis for Python 3"
   homepage "https://graph-tool.skewed.de/"
-  url "https://downloads.skewed.de/graph-tool/graph-tool-2.40.tar.bz2"
-  sha256 "d7c833c910ec0763119eb168f2f83f03dabcfaaf1a560597f9f58de0a4921130"
+  url "https://downloads.skewed.de/graph-tool/graph-tool-2.41.tar.bz2"
+  sha256 "7fb02c1891350cd2db255c000e47fa7b9af88b77f206604b926e7046f3ad7c57"
   license "LGPL-3.0"
```
Comment (Moisan) [inline Formula/graph-tool.rb:8]: The code headers include 
```
// This program is free software; you can redistribute it and/or modify it under
// the terms of the GNU Lesser General Public License as published by the Free
// Software Foundation; either version 3 of the License, or (at your option) any
// later version.
```
So the license should be
```suggestion
  license "LGPL-3.0-or-later"
```

Comment (nandahkrishna) [reply Formula/graph-tool.rb]: I've opened a separate PR.

Comment (nandahkrishna) [reply Formula/graph-tool.rb]: Oops, you beat me to it. Wanted to avoid rerunning CI.

Comment (count0) [reply Formula/graph-tool.rb]: It's fine; it had barely started.

**Review by nandahkrishna — APPROVED** [review]
Thanks, @count0!

**Pull request closed without merging**